api.md

```diff
@@ -1354,6 +1584,72 @@ enterprise | f       | 2019-02-15 13:44:53-05
 (1 row)
 ```
 
+---
+## timescaledb_information.policy_stats[](timescaledb_information-policy_stats)
+
+Show information and statistics about policies created to manage data retention and other administrative tasks on hypertables. (See policies)
+The statistics include information about when and whether the background job used to implement the policy succeeded and when it is scheduled to run next etc. 
```

Comment:
The statistics include various information, such as when and whether the background job used to implement the policy succeeded and when it is scheduled to run next.